### PR TITLE
new eve tag

### PR DESCRIPTION
**Pull request by giggsoff**
Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -32,7 +32,7 @@ require (
 	github.com/lf-edge/adam v0.0.0-20200910222414-5a4c90bf19c2
 	github.com/lf-edge/eden/eserver v0.0.0-00010101000000-000000000000
 	github.com/lf-edge/edge-containers v0.0.0-20200921124327-cb84b924624b
-	github.com/lf-edge/eve/api/go v0.0.0-20201009050143-0ca6703afd1e
+	github.com/lf-edge/eve/api/go v0.0.0-20201019011810-f8c60fefb83f
```
Comment (rvs) [inline go.mod:35]: We should probably add go.mod &co to ignore file for Yetus

File: pkg/defaults/defaults.go [hunk 2]
```diff
@@ -109,8 +104,6 @@ const (
 
 	DefaultGCPModel = "GCP"
 
-	DefaultEVEName = "node1"
```
Comment (rvs) [inline pkg/defaults/defaults.go:112]: Wait, why are we removing all this @giggsoff ?

**Review by rvs — APPROVED** [review]
LGTM but would appreciate answers to my questions.

File: go.mod (hunk 1 above)
Comment (giggsoff) [reply go.mod:35]: fixed

File: pkg/defaults/defaults.go (hunk 2 above)
Comment (giggsoff) [reply pkg/defaults/defaults.go:112]: We use them in tests/integration. These variables are not required now.

**Comment by mydatascience** [comment]
Eden test fails

**Pull request merged**